docs/en/core/math.md

```diff
@@ -1,9 +1,23 @@
 # Math
 
-Zettlr supports writing math and formulae using the KaTeX library. In order to use math, enclose your formulae with dollar signs (`$`). You can also use so-called "display math" by using two dollar signs instead of single dollar signs to enclose your formulae.
+Zettlr supports writing math and formulae using the KaTeX library. In order to use math, enclose your formulae with dollar signs (`$`). You can also use math blocks by using two dollar signs instead of a single dollar sign to enclose your formulae. On export, the double dollar signs are replaced with the LaTeX environment commands `\begin{equation}` and `\end{equation}`. Anything detected as inline or math block will receive proper MathTeX syntax highlighting.
 
-You can define math blocks by putting two dollar signs on a single line before and after a math block. Anything detected as inline or display-math will receive proper MathTeX syntax highlighting.
+```
+Everybody should know Euler's formula: $e^{ \pm i\theta } = \cos \theta \pm i\sin \theta$ 
 
-If you activate the corresponding option, Zettlr will immediately render formulae in place, making it easy for you to proofread what you typed and fix errors. If a formula contains an error, this will be shown by Zettlr.
+These are Euler's most famous equations:
+$$
+\begin{aligned}
+e^{ \pm i\theta } & = \cos \theta \pm i\sin \theta    \\
+e^{i \pi} & = -1
+\end{aligned}
+$$
+```
+
+> Please note, that no spaces must be inserted between the inline formula and the dollar signs!
```

Comment:
This wording sounds weird. Maybe something like "To prevent confusion with amounts of money, spaces between the formula and the dollar signs are not allowed."